modules/roles_profiles/manifests/profiles/files_system_managment.pp

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::files_system_managment {
```

Comment:
This seems like it should belong under a windows specific module like 'win_filesystem'.  I would create 2 classes under that such as:
win_filesystem::disable_last_access
win_filesystem::disable_8dot3

Reply:
Done.